FGFP/app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,21 @@
 
   <body>
     <%= render "layouts/header" %>
+    
+    <!-- Added from OAuth tutorial; maybe should migrate to layouts/header? -->
+    <!-- <div>
+      <% if current_user %>
+        Signed in as <strong><%= current_user.name %></strong>!
+        <%= link_to "Sign out", signout_path, id: "sign_out" %>
+      <% else %>
+        <%= link_to "Sign in with Google", "/auth/google_oauth2", id: "sign_in" %>
+      <% end %>
+    </div> -->
+
+
```

Comment:
On line 26 there is a yield (which displays the html of whatever page we are on) and then later on line 41 there is another yield. This might be why all pages are doubled.